bundles/org.openhab.core/src/main/java/org/openhab/core/library/types/QuantityType.java

```diff
@@ -249,7 +249,7 @@ public String format(String pattern) {
             if (millis != null) {
                 try {
                     return String.format(formatPattern,
-                            ZonedDateTime.ofInstant(Instant.ofEpochMilli(millis.longValue()), ZoneId.systemDefault()));
+                            ZonedDateTime.ofInstant(Instant.ofEpochMilli(millis.longValue()), ZoneId.of("UTC")));
```

Comment:
Can we make the UTC ZoneId static?

Reply:
There isn't a constants for it in the `ZoneId` class. But we can use `ZoneOffset.UTC` which is a constant and sub-class of `ZoneId`.